src/xdocs/writingchecks.xml

```diff
@@ -115,6 +115,107 @@
 
     </section>
 
+      <section name="Tool to print Javadoc tree structure">
```

Comment:
This section should be for the `Java` tree.
It should showcase `-t` and `-T` options.

This section should have nothing related to the Javadoc and it's tree and command line arguments (`-J` and `-j`).